integrations/event-handler/cli.go

```diff
@@ -242,42 +242,40 @@ func (c *StartCmdConfig) Validate() error {
 }
 
 // Dump dumps configuration values to the log
-func (c *StartCmdConfig) Dump(ctx context.Context) {
-	log := logger.Get(ctx)
-
+func (c *StartCmdConfig) Dump(ctx context.Context, log *slog.Logger) {
 	// Log configuration variables
-	log.WithField("batch", c.BatchSize).Info("Using batch size")
-	log.WithField("types", c.Types).Info("Using type filter")
-	log.WithField("skip-event-types", c.SkipEventTypes).Info("Using type exclude filter")
-	log.WithField("types", c.SkipSessionTypes).Info("Skipping session events of type")
-	log.WithField("value", c.StartTime).Info("Using start time")
-	log.WithField("timeout", c.Timeout).Info("Using timeout")
-	log.WithField("url", c.FluentdURL).Info("Using Fluentd url")
-	log.WithField("url", c.FluentdSessionURL).Info("Using Fluentd session url")
-	log.WithField("ca", c.FluentdCA).Info("Using Fluentd ca")
-	log.WithField("cert", c.FluentdCert).Info("Using Fluentd cert")
-	log.WithField("key", c.FluentdKey).Info("Using Fluentd key")
-	log.WithField("window-size", c.WindowSize).Info("Using window size")
+	log.InfoContext(ctx, "Using batch size", "batch", c.BatchSize)
+	log.InfoContext(ctx, "Using type filter", "types", c.Types)
+	log.InfoContext(ctx, "Using type exclude filter", "skip_event_types", c.SkipEventTypes)
+	log.InfoContext(ctx, "Skipping session events of type", "types", c.SkipSessionTypes)
+	log.InfoContext(ctx, "Using start time", "value", c.StartTime)
+	log.InfoContext(ctx, "Using timeout", "timeout", c.Timeout)
+	log.InfoContext(ctx, "Using Fluentd url", "url", c.FluentdURL)
+	log.InfoContext(ctx, "Using Fluentd session url", "url", c.FluentdSessionURL)
+	log.InfoContext(ctx, "Using Fluentd ca", "ca", c.FluentdCA)
+	log.InfoContext(ctx, "Using Fluentd cert", "cert", c.FluentdCert)
+	log.InfoContext(ctx, "Using Fluentd key", "key", c.FluentdKey)
+	log.InfoContext(ctx, "Using window size", "window_size", c.WindowSize)
 
 	if c.TeleportIdentityFile != "" {
-		log.WithField("file", c.TeleportIdentityFile).Info("Using Teleport identity file")
+		log.InfoContext(ctx, "Using Teleport identity file", "file", c.TeleportIdentityFile)
 	}
 	if c.TeleportRefreshEnabled {
-		log.WithField("interval", c.TeleportRefreshInterval).Info("Using Teleport identity file refresh")
+		log.InfoContext(ctx, "Using Teleport identity file refresh", "interval", c.TeleportRefreshInterval)
 	}
 
 	if c.TeleportKey != "" {
-		log.WithField("addr", c.TeleportAddr).Info("Using Teleport addr")
-		log.WithField("ca", c.TeleportCA).Info("Using Teleport CA")
-		log.WithField("cert", c.TeleportCert).Info("Using Teleport cert")
-		log.WithField("key", c.TeleportKey).Info("Using Teleport key")
+		log.InfoContext(ctx, "Using Teleport addr", "addr", c.TeleportAddr)
+		log.InfoContext(ctx, "Using Teleport CA", "ca", c.TeleportCA)
+		log.InfoContext(ctx, "Using Teleport cert", "cert", c.TeleportCert)
+		log.InfoContext(ctx, "Using Teleport key", "key", c.TeleportKey)
```

Comment:
nit: create a `teleport` slog group?

Reply:
I kinda wonder if there may be a simpler way of dumping the config.

Reply:
I like this suggestion muchly

Reply:
```
(devbox) ➜  event-handler git:(strideynet/convert-event-handler-to-slog) ✗ ./build/teleport-event-handler start --config tmp/teleport-event-handler.toml --start-time 2020-01-01T00:00:00Z -d
 INFO  Initializing event handler enabled:false batch:20 types:[] skip_event_types:[] skip_session_types:[print] timeout:10s dry_run:false exit_on_last_event:false concurrency:5 window_size:24h0m0s config.fluentd.url:https://localhost:8888/test.log config.fluentd.session_url:https://localhost:8888/session config.fluentd.cert:/Users/noah/code/gravitational/teleport/integrations/event-handler/tmp/client.crt config.fluentd.key:/Users/noah/code/gravitational/teleport/integrations/event-handler/tmp/client.key config.fluentd.ca:/Users/noah/code/gravitational/teleport/integrations/event-handler/tmp/ca.crt config.teleport.addr:localhost:3025 config.teleport.identity_file:/Users/noah/code/gravitational/teleport/integrations/event-handler/tmp/identity refresh_enabled:true refresh_interval:1m0s config.teleport.ca: config.teleport.cert: config.teleport.key: event-handler/app.go:163
```

Sadly our current formatter tends to mangle group keys.